src/actions/createGame.js

```diff
@@ -0,0 +1,20 @@
+import createRoom from '../socketHandlers/createRoom';
+import socket from '../socketHandlers/index';
+
+const createGame = () => {
+  return (dispatch) => {
+    createRoom(() => {
+      const roomId = socket.id;
+      return dispatch(createGameSuccess(roomId));
```

Comment:
Maybe you can delete this `return` statement.

Reply:
@duyoji 
You mean both line 5 and 8?